auth/handlers/login.go

```diff
@@ -0,0 +1,36 @@
+package handlers
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+)
+
+// MakeLoginHandler creates a handler for logging in
+func MakeLoginHandler(config *Config) func(http.ResponseWriter, *http.Request) {
+	return func(w http.ResponseWriter, r *http.Request) {
+		fmt.Println("Path", r.URL.Path)
+
+		resource := "/"
+
+		if val := r.URL.Query().Get("r"); len(val) > 0 {
+			resource = val
+		}
+
+		if strings.EqualFold(r.URL.Path, "/login/github") {
```

Comment:
Does `/login/github/` need to be valid as well?

Reply:
I'm missing something. `if strings.EqualFold(r.URL.Path, "/login/github")` this checks for the r.URL.Path to be exactly equal to `/login/github` right?